k8s/examples/vineyard-csidriver/Dockerfile

```diff
@@ -1,12 +1,10 @@
 FROM python:3.10
 
-RUN pip3 install --no-cache-dir pandas requests scikit-learn numpy vineyard
+RUN pip3 install pandas requests scikit-learn numpy vineyard
```

Comment:
Use `--no-cache-dir` to let pip cleanup the downloaded zips/wheels after installation.

Reply:
Done.